### ISSUE TITLE
cmd, load requests failing with CLIv2: initiate request failed

### ISSUE THREAD

**Issue by g24swint** [capitalone/giraffez#23]
When executing back-to-back requests, particularly pretty quickly, Giraffez comments can fail with error message "TeradataError: CLIv2: initiate request failed"

Versions
- python 3.5.2 from a conda environment, with 
- giraffez 1.1.5 (pip installed to the conda environment, without --user)
- teradata client library 15.00, 64-bit

I have noticed that sometimes executing requests back-to-back can lead to a first one to succeed and a second to fail; and further that if the cmd calls were split between two jupyter notebook cells then both would succeed.

```
---------------------------------------------------------------------------
error                                     Traceback (most recent call last)
.../python/conda_envs/py3k/lib/python3.5/site-packages/giraffez/cmd.py in _execute(self, command)
    105         try:
--> 106             results, error = self.cmd.execute(command)
    107             if error is not None:

error: CLIv2: initiate request failed

During handling of the above exception, another exception occurred:

TeradataError                             Traceback (most recent call last)
<ipython-input-23-60624f138025> in <module>()
      2 # a simple giraffez command execution
      3 with gr.Cmd() as cmd:
----> 4     results = cmd.execute_one('select * sample 5 from padw.pl_appn_root')
      5 samp_data = pd.DataFrame(results.items())
      6 samp_data

.../python/conda_envs/py3k/lib/python3.5/site-packages/giraffez/cmd.py in execute_one(self, command, sanitize, silent)
    210         :raises `giraffez.errors.GiraffeError`: if the return data could not be decoded
    211         """
--> 212         return self.execute(command, sanitize, silent).one()
    213 
    214     def execute_many(self, command, sanitize=True, parallel=False, silent=False):

.../python/conda_envs/py3k/lib/python3.5/site-packages/giraffez/cmd.py in execute(self, command, sanitize, silent)
    196             command = prepare_statement(command) # accounts for comments and newlines
    197             log.debug("Debug[2]", "Command (sanitized): {!r}".format(command))
--> 198         return Results(self._execute(command))
    199 
    200     def execute_one(self, command, sanitize=True, silent=False):

.../python/conda_envs/py3k/lib/python3.5/site-packages/giraffez/cmd.py in _execute(self, command)
    109             log.debug("Debug[2]", repr(results))
    110         except _cli.error as error:
--> 111             raise TeradataError(error)
    112         for result in results:
    113             stmt_info = result.get('stmtinfo', None)

TeradataError: CLIv2: initiate request failed
```

**Comment by g24swint** [comment]
Per an offline conversation, I did pull the current master version of giraffez which has 1.1.6 changes merged in, pip uninstalled 1.1.5, and python setup.py develop installed 1.1.6 from the repo, and the issue does seem to be resolved in that version. 

BONUS: this also seemed to resolve an error with load which had been failing after a few thousand records

**Comment by ChrisRx** [comment]
Closing this issue down since it has been addressed in a previous commit.  I can reopen if the problem persists.
